website/content/en/docs/main/tutorials/iperf/index.md

```diff
@@ -88,168 +78,32 @@ Install iPerf3 (client and server) on the clusters:
 *Client cluster*:
 
 ```sh
-export IPERF3_FILES=https://raw.githubusercontent.com/clusterlink-net/clusterlink/main/demos/iperf3/testdata/manifests
-kubectl apply -f $IPERF3_FILES/iperf3-client/iperf3-client.yaml
+export TEST_FILES=https://raw.githubusercontent.com/clusterlink-net/clusterlink/main/demos/iperf3/testdata/manifests
+kubectl apply -f $TEST_FILES/iperf3-client/iperf3-client.yaml
 ```
 
 *Server cluster*:
 
 ```sh
-export IPERF3_FILES=https://raw.githubusercontent.com/clusterlink-net/clusterlink/main/demos/iperf3/testdata/manifests
-kubectl apply -f $IPERF3_FILES/iperf3-server/iperf3.yaml
+export TEST_FILES=https://raw.githubusercontent.com/clusterlink-net/clusterlink/main/demos/iperf3/testdata/manifests
+kubectl apply -f $TEST_FILES/iperf3-server/iperf3.yaml
 ```
 
 ## Deploy ClusterLink
 
-1. Create the fabric and peer certificates for the clusters:
-
-    *Client cluster*:
-
-    ```sh
-    clusterlink create fabric
-    clusterlink create peer-cert --name client
-    ```
-
-    *Server cluster*:
-
-    ```sh
-    clusterlink create peer-cert --name server
-    ```
-
-    For more details regarding fabric and peer see [core concepts][].
-
-1. Deploy ClusterLink on each cluster:
-
-    *Client cluster*:
-
-    ```sh
-    clusterlink deploy peer --name client --ingress=NodePort --ingress-port=30443
-    ```
-
-    *Server cluster*:
-
-    ```sh
-    clusterlink deploy peer --name server --ingress=NodePort --ingress-port=30443
-    ```
-
-   {{< notice note >}}
-   This tutorial uses NodePort to create an external access point for the kind clusters.
-    By default `deploy peer` creates an ingress of type LoadBalancer,
-    which is more suitable for Kubernetes clusters running in the cloud.
-   {{< /notice >}}
-
-1. Verify that ClusterLink control and data plane components are running:
-
-   It may take a few seconds for the deployments to be successfully created.
-
-   *Client cluster*:
-
-   ```sh
-   kubectl rollout status deployment cl-controlplane -n clusterlink-system
-   kubectl rollout status deployment cl-dataplane -n clusterlink-system
-   ```
-
-   *Server cluster*:
-
-   ```sh
-   kubectl rollout status deployment cl-controlplane -n clusterlink-system
-   kubectl rollout status deployment cl-dataplane -n clusterlink-system
-   ```
-
-   {{% expand summary="Sample output" %}}
-
-   ```sh
-   deployment "cl-controlplane" successfully rolled out
-   deployment "cl-dataplane" successfully rolled out
-   ```
-
-   {{% /expand %}}
+{{% include "../../../../static/files/tutorials/deploy-clusterlink.md" %}}
```

Comment:
why using `../../....../static/files` instead of just `/static` like in other places?
Hugo knows to search the path and using relative parent directories is prone to errors if this file's location changes.